packages/nextcloud/lib/src/version_supported.dart

```diff
@@ -1,33 +1,57 @@
 part of '../nextcloud.dart';
 
+/// Version of core/Server supported
+const coreSupportedVersion = 26;
+
+/// API version of the news app supported
+const newsSupportedVersion = 'v1-3';
+
+/// API version of the notes app supported
+const notesSupportedVersion = 1;
+
+/// All supported versions
+const supportedVersions = [
+  coreSupportedVersion,
+  newsSupportedVersion,
+  notesSupportedVersion,
+];
+
 // ignore: public_member_api_docs
 extension CoreVersionSupported on NextcloudCoreClient {
   /// Checks if the app on the server is supported by the client
-  Future<bool> isSupported([final NextcloudCoreServerCapabilities_Ocs_Data? capabilities]) async =>
-      (capabilities ?? (await getCapabilities()).ocs.data).version.major == 26;
+  Future<(bool, int)> isSupported([
```

Comment:
The version that is returned is always just the constant and could be queried separately, but I think this interface makes it a little easier to work with.

Reply:
Yeah but we don't hide it so if somebody wants to look it up separately they can.